estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_view_list" model="ir.ui.view">
+        <field name="name">estate.property.list</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <list string="Properties">
+                <field name="name"/>
+                <field name="property_type_id" optional="show"/>
+                <field name="postcode" optional="show"/>
+                <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="show"/>
+                <field name="bedrooms" optional="hide"/>
+                <field name="living_area" optional="hide"/>
+                <field name="expected_price" optional="show"/>
+                <field name="selling_price" optional="show"/>
+                <field name="date_availability" optional="hide"/>
+                <field name="state" widget="badge"
+                       decoration-info="state == 'offer_received'"
+                       decoration-success="state in ['sold', 'offer_accepted']"
+                       decoration-danger="state == 'canceled'"
+                       optional="show"/>
+            </list>
+        </field>
+    </record>
+
+    <record id="estate_property_view_form" model="ir.ui.view">
+        <field name="name">estate.property.form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Properties">
+                <header>
+                    <button name="action_sold" type="object" class="btn-primary" string="Sold"
+                            invisible="state in ['sold', 'canceled']"/>
+                    <button name="action_canceled" type="object" string="Cancel"
+                            invisible="state in ['sold', 'canceled']"/>
+                    <button name="action_reset" type="object" string="Reset"
+                            invisible="state not in ['sold', 'canceled']"/>
+                    <field name="state" widget="statusbar" statusbar_visible="new,offer_received,offer_accepted,sold"/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+
+                    </div>
```

Comment:
It is here to let modules add box buttons without stepping on each other.
Ex: If 2 modules add buttons they already have the parent div, and do not need to step on each other to add it

Reply:
That is good thinking if you know it will be used for inheritance ! But usually you never know if that will be the case, and we avoid adding extra nodes like this unless we know this part will particularly be inherited multiple times. Or if it can't be done otherwise, but here you can use `expr="//div[@class='oe_title']"` in your inheriting view to get it.

In this case I insists because I don't think an extra `<div class="oe_button_box" name="button_box">` is needed around the oe-stat button, but if it is, you can eventually leave it like this